fairing/preprocessors/converted_notebook.py

```diff
@@ -65,16 +65,20 @@ def __init__(self,
 
         self.notebook_file = notebook_file
         self.notebook_preprocessor = notebook_preprocessor
+        self.not_overwrite = False
 
     def preprocess(self):
+        if self.not_overwrite:
```

Comment:
Can you add a test case for the new code here? Maybe just passing in various flag in the tests here:

https://github.com/kubeflow/fairing/blob/master/tests/unit/preprocessors/test_full_notebook_preprocessor.py#L12

Reply:
Sure, I will add a test case for the flag.